risc0/zkvm/lib.rs

```diff
@@ -12,34 +13,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use anyhow::Result;
-use cxx::{let_cxx_string, UniquePtr};
+use anyhow::{Error, Result};
+use cxx::let_cxx_string;
 
-#[cxx::bridge(namespace = "risc0::rust")]
-mod ffi {
+
+#[cxx::bridge(namespace = "risc0")]
+pub mod ffi {
     unsafe extern "C++" {
         include!("risc0/zkvm/prove/method_id.h");
 
-        type MethodID;
-
-        fn new_method_id(elf_path: &CxxString) -> Result<UniquePtr<MethodID>>;
-        fn write(&self, filename: &CxxString) -> Result<()>;
+        #[cxx_name = "makeMethodId"]
+        fn make_method_id_from_elf(path: &CxxString) -> Result<[u8; 384]>;
```

Comment:
Use `METHOD_ID_LEN`

Reply:
CXX won't allow it...but at least this will break at compile time if the two constants get out of whack